pkg/crd/generator/testData/pkg/apis/fun/v1alpha1/toy_types.go

```diff
@@ -76,6 +76,13 @@ type ToySpec struct {
 	// This is an IPv4 address.
 	// +kubebuilder:validation:Format=ipv4
 	Address string `json:"address"`
+
+	// This is a list of IPv4 addresses.
+	// +kubebuilder:validation:Format=ipv4
+	Addresses []string `json:"addresses"`
+
+	// This is a byte array.
+	Bytes []byte `json:"bytes"`
 }
```

Comment:
I just realized that "Comment" on line 83 is also a []byte array... i will remove this one.